### ISSUE TITLE
Spoolman Dialog Shows Wrong Times

### ISSUE THREAD

**Issue by Ocraftyone** [fluidd-core/fluidd#1178]
### Fluidd Version

v1.25.3-26d94c8

### Browser

Firefox

### Device

Desktop PC

### Operating System

Windows

### What happened

The spoolman dialog does not take into account timezones. I have both the timezone on my pi and the timezone in the spoolman docker container set to the same zone. Spoolman seems to store the times in UTC then converts the time on the fly when displaying to the user (as evident by when you edit the spool it is in UTC and when just viewing the time it is converted to the proper timezone). Fluidd just needs to take into account the timezone before displaying when it was last used.

Fluidd Spoolman Dialog:
![image](https://github.com/fluidd-core/fluidd/assets/24759591/2b0b49a2-ba43-4aa3-9c25-6ee1e0815214)

Native Spoolman Page:
![image](https://github.com/fluidd-core/fluidd/assets/24759591/ee57c005-ff21-490b-8872-3ae9b76d2ece)

Under Edit Spool in native spoolman page:
![image](https://github.com/fluidd-core/fluidd/assets/24759591/9facf207-0a4a-4e87-b9c0-257adc5e52cd)

### What did you expect to happen

The timezone is taken into account when calculating when the spool was last used.

### How to reproduce

Use a spoolman instance and fluidd instance with timezones

**Comment by pedrolamas** [comment]
Hi @Ocraftyone thank you for reporting this.

From a quick check on Fluidd code, I believe we are using UTC dates and assuming that is what Spoolman API is giving us - so that will be the first thing to check.

We will report back when we know more.

**Comment by pedrolamas** [comment]
I have now confirmed and reproduced this issue, though I would argue this is a Spoolman API issue, not a Fluidd one!

As you well said and I confirmed in the Spoolman docs, the Spoolman API returns dates & times in UTC.

However, the string representation of said dates & times is missing the time zone indicator as per [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).

For example, calling `/api/v1/spool`, we get a list of spools and I can see the data in all date fields have no time zone indication:

![image](https://github.com/fluidd-core/fluidd/assets/85504/28551a82-ea3b-455d-9d08-ee70b856f9bd)

As these are all UTC dates & times, these strings should end with a single "Z" - following the ISO and thus solving this issue in Fluidd and any other frontend.

I will open an issue with Spoolman regarding this and see if they want to fix it upstream as I believe it should be.
